### ISSUE TITLE
Add support for ebpf tcptop metrics

### ISSUE THREAD

**Issue by maitreya-source** [williamchanrico/planet-exporter#6]
* What and how?
  * [eBPF](https://ebpf.io/) can be used to extract telemetry data from Linux kernel. For more info on eBPF - https://ebpf.io/
  * [eBPF-exporter](https://github.com/cloudflare/ebpf_exporter) is a prometheus exporter for custom eBPF metrics. We can define our own configuration similar to [examples](https://github.com/cloudflare/ebpf_exporter/tree/master/examples) here.
  *  A config can be created to extract 5 tuple network telemetry information similar to [bcc/tcptop.py](https://github.com/iovisor/bcc/blob/master/tools/tcptop.py).
  *  The prometheus output of eBPF exporter tcptop can be read and enriched by PE.

* Why?
   * In short term, we can get telemetry data for total bytes sent and received. This data will be extracted directly by attaching kprobes to kernel tcp functions.
   * In long term, we can add support for more data that can be extracted from eBPF and enriched/processed in PE.

Please feel free to assign this issue to me. :)

**Comment by williamchanrico** [comment]
Hi @maitreya-source, Thanks for raising this issue!
This is a great addition that could increase the overall performance and reduce the complexity of darkstat coupling today.

**Comment by maitreya-source** [comment]
I'll close this once #13 is merged.
